src/Controls/tests/DeviceTests/Elements/Modal/ModalTests.iOS.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Handlers;
+using UIKit;
+using Xunit;
+
+namespace Microsoft.Maui.DeviceTests
+{
+	public partial class ModalTests : ControlsHandlerTestBase
+	{
+		[Theory]
+		[ClassData(typeof(PageTypes))]
+		public async Task PushModalUsingTransparencies(Page rootPage, Page modalPage)
+		{
+			SetupBuilder();
+
+			var expected = Colors.Red;
+
+			rootPage.BackgroundColor = expected;
+			modalPage.BackgroundColor = Colors.Transparent;
+
+			await CreateHandlerAndAddToWindow<IWindowHandler>(rootPage,
+				async (handler) =>
+				{
+					var currentPage = (rootPage as IPageContainer<Page>).CurrentPage;
+					await currentPage.Navigation.PushModalAsync(modalPage);
+					await OnLoadedAsync(modalPage);
+					Assert.Equal(1, currentPage.Navigation.ModalStack.Count);
+
+					var rootView = handler.PlatformView;
+					Assert.NotNull(rootView);
```

Comment:
I think here you need to test that `rootView.Window` is not null

Reply:
Changed.

Reply:
Did you push up your changes? I don't see any recent commits and the code still looks the same.